docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -2435,9 +2435,17 @@ alertmanager_client:
 
 # (experimental) Enable UTF-8 strict mode. Allows UTF-8 characters in the
 # matchers for routes and inhibition rules, in silences, and in the labels for
-# alerts. It is recommended to check both alertmanager_matchers_disagree_total
-# and alertmanager_matchers_incompatible_total metrics before using this mode as
-# otherwise some tenant configurations might fail to load.
+# alerts. It is recommended that all tenants run the `migrate-utf8` command in
+# mimirtool before enabling this mode as otherwise some tenant configurations
+# might fail to load. Tenants with incompatible configurations can be identified
+# in Mimir server logs searching for lines containing `Alertmanager is moving to
+# a new parser for labels and matchers, and this input is incompatible.`. Tenant
```

Comment:
```suggestion
# mimirtool before enabling this mode. Otherwise, some tenant configurations
# might fail to load. To identify tenants with incompatible configurations, you can search
# Mimir server logs for lines containing `Alertmanager is moving to
# a new parser for labels and matchers, and this input is incompatible.`.
```